### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by Tresor11**
On this branch, we added some styles to the app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
+- React
+- Redux
+- Node js
+- Heroku
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://book-store-t.herokuapp.com/)
```
Comment (bolah2009) [inline README.md:19]: The live link seems to be showing an empty, page. Let's fix this.

https://book-store-t.herokuapp.com/

image

File: src/containers/BooksForm.js [hunk 2]
```diff
@@ -31,13 +31,14 @@ class BooksForm extends React.Component {
...
+        <h4 className="form-control new-book-text">ADD NEW BOOK</h4>
+        <form className="form-control">
+          <input type="text" className="w-50 h-40" required placeholder="Book title" name="title" value={this.state.title} onChange={this.handleChange} />
+          <select name="category" className="w-25 h-40" value={this.state.category} onChange={this.handleChange}>
+            <option value="">Category</option>
             {categories.map(el => <option key={Math.random() * 100} value={el}>{el}</option>)}
           </select>
-          <button type="submit" onClick={this.handleSubmit}>Submit</button>
+          <button type="submit" className="w-20 h-40 blue" onClick={this.handleSubmit}>Add Book</button>
         </form>
```
Comment (bolah2009) [inline src/containers/BooksForm.js]: [Optional]

Why not use the `onSubmit` on the `form` element instead, this way you get the native HTML5 form validation by default when you add the `required` attribute to each `input/select` element? 🤔 

See demo below where an empty book is added to the store, I don't think this should be the expected behaviour of the app.

record1

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Awesome job team!!! 👏🏾👏🏾👏🏾

The design looks good to me. However, can you please fix the issues listed below:
- Add working a  live link to the readme (see comments below).
- Remove unnecessary disabled ESLint rules (see https://github.com/Tresor11/BookStore/pull/3#pullrequestreview-416026269 for more details).
Examples are as follows:
   - https://github.com/Tresor11/BookStore/blob/fc01b3ff33a80f21f238eac1e3d052c3f1e653c3/src/containers/BooksForm.js#L1 
   - https://github.com/Tresor11/BookStore/blob/fc01b3ff33a80f21f238eac1e3d052c3f1e653c3/src/components/Book.js#L1
   - https://github.com/Tresor11/BookStore/blob/fc01b3ff33a80f21f238eac1e3d052c3f1e653c3/src/actions/index.js#L1
   - https://github.com/Tresor11/BookStore/blob/fc01b3ff33a80f21f238eac1e3d052c3f1e653c3/src/index.js#L1
   - https://github.com/Tresor11/BookStore/blob/fc01b3ff33a80f21f238eac1e3d052c3f1e653c3/src/serviceWorker.js#L1 You could delete this file since you not using it.

#### :recycle:  Make these changes and submit another code review request. 👍🏾

#### Feel free to reach me on slack if you having issues implementing these features. :relaxed: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by fatymahmed — APPROVED** [review]
## Hi @Tresor11 @Datagirlcmr 
 Great job making the required changes 🎉🎉
## ✔️ Approved!

**Pull request merged**